client/app/telecom/telephony/alias/configuration/changeType/telecom-telephony-alias-configuration-changeType.html

```diff
@@ -1,104 +1,127 @@
-                                required
-                                data-ng-model="AliasConfigurationChangeTypeCtrl.number.feature.featureType"
-                                data-ng-options="type.id as type.label for type in AliasConfigurationChangeTypeCtrl.availableTypes | orderBy:'label'">
-                                <option value="" data-translate="telephony_alias_config_change_type_label_select" disabled></option>
-                        </select>
-                    </div>
-
-                    <telephony-bulk-action data-service-type="alias"
-                                           data-billing-account="{{ AliasConfigurationChangeTypeCtrl.number.billingAccount }}"
-                                           data-service-name="{{ AliasConfigurationChangeTypeCtrl.number.serviceName }}"
-                                           data-custom-class="link d-block pl-0 mb-3"
-                                           data-bulk-infos="AliasConfigurationChangeTypeCtrl.bulkDatas.infos"
-                                           data-ng-disabled="!changeTypeForm.$valid"
-                                           data-get-bulk-params="AliasConfigurationChangeTypeCtrl.getBulkParams"
-                                           data-on-success="AliasConfigurationChangeTypeCtrl.onBulkSuccess"
-                                           data-on-error="AliasConfigurationChangeTypeCtrl.onBulkError">
-                    </telephony-bulk-action>
-
-                    <button type="submit"
-                            class="btn btn-primary"
-                            data-ng-disabled="!changeTypeForm.$valid || AliasConfigurationChangeTypeCtrl.loading.changing || !AliasConfigurationChangeTypeCtrl.number.feature.hasChange()"
-                            data-translate="submit">
-                    </button>
-                </form>
-                <div class="text-center"
-                     data-ng-if="AliasConfigurationChangeTypeCtrl.loading.changing">
-                    <oui-spinner></oui-spinner>
-                    <h4 data-translate="telephony_alias_config_change_type_doing"></h4>
-                </div>
+        <div class="row w-100 d-md-flex">
+            <div class="col-lg-4 col-sm-12 mb-4">
+                <oui-select-picker class="h-100 pr-0 ml-0"
+                                   data-match="name"
+                                   data-model="$ctrl.selectedFeatureType"
+                                   data-label="{{:: 'telephony_alias_config_change_type_label_cloudIvr' | translate }}"
+                                   data-values="[{ id: 'cloudIvr', name: $ctrl.$translate.instant('telephony_alias_config_change_type_desc_cloudIvr') }]"
+                                   data-variant="light">
+                    <span class="transclude-image-container">
+                        <img src="assets/images/aliasFeature/schema-cloudIvr.svg"
+                             alt="schema-cloudIvr.svg">
```

Comment:
Could be more descriptive?

Reply:
https://github.com/ovh-ux/ovh-manager-telecom/pull/981#discussion_r225066859